apps/ima/translations/src/en/fields.json

```diff
@@ -331,6 +331,45 @@
   "permission-to-enter-or-stay-detail": {
     "label": "Explain what permission you have"
   },
+  "is-serious-and-irreversible": {
+    "legend": "Does your Illegal Migration Act notice say that you can make a serious and irreversible harm claim?",
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "is-risk-in-country":{
+    "legend": "Would you be at risk?",
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "reason-in-sih": {
+    "label": "Why would you be at risk?"
+  },
+  "why-not-get-protection": {
+    "label":"Why would you not be able to get protection from this harm in {{values.country-1}}?",
+    "summary-heading1":"Why would you not be able to get protection from this harm in {{country-name-1}}?",
+    "summary-heading2":"Why would you not be able to get protection from this harm in {{country-name-2}}?",
+    "summary-heading3":"Why would you not be able to get protection from this harm in {{country-name-3}}?",
+    "summary-heading4":"Why would you not be able to get protection from this harm in {{country-name-4}}?",
+    "summary-heading5":"Why would you not be able to get protection from this harm in {{country-name-5}}?"
```

Comment:
Is it possible to use one message dynamically for this purpose, rather than using the switch to get one of these messages can we inject any country name into a single generic message?

Reply:
It works works only with these messages